app/services/moderator/banish_user.rb

```diff
@@ -19,6 +19,7 @@ def banish
       delete_user_activity
       delete_comments
       delete_articles
+      cleanup_chat_channels
```

Comment:
Not sure if we really need this level of indirection, I was just following the established pattern here. Will we also be calling delete_chat_channels in other contexts than banishing users?

Reply:
I think there aren't other contexts where we'd deleted chat channels; maybe an opportunity to refactor in a different PR?

Reply:
Since I just added this method, I'd rather update this PR and move it right away.